src/Messages.js

```diff
@@ -62,6 +62,31 @@ export default defineMessages({
         description: 'Vulnerabilities - vulnerability total',
         defaultMessage: '{total}'
     },
+    last90days: {
```

Comment:
rather than doing three of the same, what if we try something like this... would be useful for others possibly

    lastxdays: {
        id: 'lastxdays',
        description: 'Last x days label where x is a predefined value',
        defaultMessage: 'Last {days} days'
    },

Reply:
great catch :+1:  I will update it